config/settings.py

```diff
@@ -409,3 +409,13 @@ def sentry_tracer_sampler(sampling_context):
 WORKSPACE_DATABASE_PASSWORD = os.environ.get("WORKSPACE_DATABASE_PASSWORD")
 WORKSPACE_DATABASE_HOST = os.environ.get("WORKSPACE_DATABASE_HOST")
 WORKSPACE_DATABASE_PORT = os.environ.get("WORKSPACE_DATABASE_PORT")
+
+# Filesystem configuration
+WORKSPACE_BUCKET_PREFIX = os.environ.get("WORKSPACE_BUCKET_PREFIX", "hexa-test-")
+WORKSPACE_BUCKET_REGION = os.environ.get("WORKSPACE_BUCKET_REGION", "EUROPE-WEST1")
```

Comment:
Are you sure about `EUROPE-WEST1`? In the docs, regions are lowercased (https://cloud.google.com/compute/docs/regions-zones)

Reply:
And I wouldn't put a default value here - rather in the Docker compose file